### PR TITLE
[CUR-667] User created at date filter and sorting

### PR DESCRIPTION
**Pull request by ahmad-curriki**
- Sort and filter by user registration dates

### CODE REVIEW

File: public/assets/js/custom.js [hunk 1]
```diff
@@ -222,6 +222,26 @@ function initializeSelect2(target, url, textProp = ['title'], idProp = 'id') {
     });
 }
 
+/**
+ * Initialize the date range picker
+ * @param target
+ */
+function initializeDateRange(target){
```
Comment (aqeel-tkxel) [inline public/assets/js/custom.js]: Space after closing ) ?

**Review by aqeel-tkxel — CHANGES_REQUESTED** [review]
Minor formatting fix required.

File: public/assets/js/custom.js (hunk 1 above)
Comment (ahmad-curriki) [reply public/assets/js/custom.js]: Closing of what?

**Review by aqeel-tkxel — APPROVED** [review]
PR Approved.

**Pull request merged**